libnavui-voice/api/current.txt

```diff
@@ -25,7 +31,7 @@ package com.mapbox.navigation.ui.voice.api {
   }
 
   public abstract sealed class VoiceInstructionsPlayerAttributes {
-    method protected abstract kotlin.jvm.functions.Function1<android.media.AudioFocusRequest.Builder,kotlin.Unit> configureAudioFocusRequestBuilder();
+    method protected abstract kotlin.jvm.functions.Function1<android.media.AudioFocusRequest.Builder,kotlin.Unit> configureAudioFocusRequestBuilder(com.mapbox.navigation.ui.voice.model.AudioFocusOwner owner);
```

Comment:
Can be skipped `Inheritance of sealed classes or interfaces from different module is prohibited`